mindsdb/integrations/handlers/statsforecast_handler/statsforecast_handler.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+import dill
+from mindsdb.integrations.libs.base import BaseMLEngine
+from statsforecast import StatsForecast
+from statsforecast.models import AutoARIMA
+
+DEFAULT_FREQUENCY = "D"
+DEFAULT_MODEL = AutoARIMA()
+DEFAULT_MODEL_NAME = "AutoARIMA"
+
+
+class StatsForecastHandler(BaseMLEngine):
+    """Integration with the Nixtla StatsForecast library for
+    time series forecasting with classical methods.
+    """
+
+    name = "statsforecast"
+
+    def create(self, target, df=None, args={}):
+        """Create the StatsForecast Handler.
+
+        Requires specifying the target column to predict and time series arguments for
+        prediction horizon, time column (order by) and grouping column(s).
+
+        Does not require a dataframe, because model fitting takes place jointly with
+        forecasting in the "predict step".
+        """
+        time_settings = args["timeseries_settings"]
+        assert time_settings["is_timeseries"], "Specify time series settings in your query"
+        ###### store model args and time series settings in the model folder
+        model_args = {}
+        model_args["target"] = target
+        model_args["horizon"] = time_settings["horizon"]
+        model_args["order_by"] = time_settings["order_by"]
+        model_args["group_by"] = time_settings["group_by"]
+
+        ###### persist changes to handler folder
+        self.model_storage.file_set("model_args", dill.dumps(model_args))
```

Comment:
Probably better to use `json_set` so that you can avoid the `dill` call.

Reply:
I've changed this now for the "model_args" storage, which is easily json-ifiable.

However, I've made dill calls for storing more complex objects. Hope this is ok, but I can try to find another solution if you'd rather avoid dill entirely.